pkgs/applications/misc/jp2a/default.nix

```diff
@@ -1,20 +1,37 @@
-{ lib, stdenv, fetchFromGitHub, libjpeg, autoreconfHook }:
+{ lib
+, stdenv
+, fetchFromGitHub
+, libjpeg
+, libpng
+, ncurses
+, autoreconfHook
+, autoconf-archive
+, pkg-config
+, bash-completion
+}:
 
 stdenv.mkDerivation rec {
-  version = "1.0.7";
+  version = "1.1.0";
   pname = "jp2a";
 
   src = fetchFromGitHub {
-    owner = "cslarsen";
+    owner = "Talinx";
```

Comment:
Why did we switch to the fork? Upstream does not redirect there.

Reply:
"Upstream" has not had an update in about four years and more importantly, this is also what [Ubunutu packages](https://launchpad.net/ubuntu/+source/jp2a).

Reply:
launchpads UI is great and I have no idea how to get the upstream repo there.